bark/bark.py

```diff
@@ -218,6 +280,40 @@ def write_metadata(filename, **params):
         yaml_file.write(yaml.safe_dump(params, default_flow_style=False))
 
 
+def _enforce_units(params):
+    """
+    Enforces strict adherence to the BARK spec for the 'units' attribute.
+
+    Modifies the given dict in place.
+    """
+    if 'units' in params:
+        if params['units'] == 'seconds':
+            params['units'] = 's'
+    return
+
+def _enforce_datatype(params):
+    """
+    Checks params['datatype'] against canonical list.
+
+    Throws KeyError if params['datatype'] is not on the list.
+
+    If there is no 'datatype' attribute, infers time series / point process
+    identity from 'units' and assigns either 0/UNDEFINED or 1000/EVENT,
+    respectively.
+
+    Modifies the given dict in place.
+    """
+    if 'datatype' in params:
```

Comment:
`datatype` is an optional attribute, I think we should leave it unassigned if it does not exist